### ISSUE TITLE
Options Menu: plain with text and plain with text disabled inaccessible by keyboard

### ISSUE THREAD

**Issue by jessiehuff** [patternfly/patternfly-react#3766]
**Describe the issue. What is the expected and unexpected behavior?**

When navigating by keyboard, you are able to open the dropdown menus of the plain with text and plain with text disabled examples with the enter or space key, but once this menu is open, there is no way to navigate to or interact with the menu items.

**Please provide the steps to reproduce. Feel free to link CodeSandbox or another tool.**
You are able to navigate to the next item after the dropdown with tab, but you are unable to enter the menu: 
<img width="370" alt="Screen Shot 2020-02-18 at 10 22 53 AM" src="https://user-images.githubusercontent.com/28281340/74749814-bbc33480-5238-11ea-9872-a1e5cf3117eb.png">

**Is this a bug or enhancement? If this issue is a bug, is this issue blocking you or is there a work-around?**
This is an accessibility bug.
